float32 ori_x
float32 ori_y
float32 ori_z
---
float32 qw
float32 qx
float32 qy
float32 qz
